frontend/src/components/invoice/models/InvoiceModel.ts

```diff
@@ -70,7 +70,7 @@ export default class InvoiceModel implements IAttachment {
     this.orderNr = obj.orderNr || '';
     this.verified = obj.verified || false;
     this.discount = obj.discount;
-    this.attachments = obj.attachments || [{type: 'pdf'}];
+    this.attachments = obj.attachments || [{type: 'pdf'}, {type:'xml', desc:'PEPPOL xml'}];
```

Comment:
Ah cool, voeg je ook een desc toe voor de pdf? "Factuur pdf" misschien?

Ik vraag me ook af of het op het "add attachment"-modal nu ook duidelijk is (vb de foutboodschap dat een bepaald type reeds aanwezig is)
Kun je dat eens checken?

Reply:
Ik heb dit even moeten uitspitten - de types die je kan uploaden via de AddAttachmentPopup worden (als ik het goed heb begrepen) bepaald door de attachmentTypes die je toevoegt aan de config. In mijn lokale versie is dat veld gewoon leeg, en kan ik daar ook geen types aan toevoegen.

`export const AddAttachmentPopup = EnhanceWithClaim(connect((state: ConfacState) => ({
  attachmentTypes: state.config.attachmentTypes,
}), {})(AddAttachmentPopupComponent));`

Reply:
Die `connect` is oude class component syntax btw
Tegenwoordig kan je dat gewoon doen met `useContext` in function components